metacat-connector-druid/src/main/java/com/netflix/metacat/connector/druid/DruidConfigConstants.java

```diff
@@ -0,0 +1,141 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.connector.druid;
+
+/**
+ * Druid Config Constants.
+ * @author zhenl
+ * @since 1.1.0
+ */
+public final class DruidConfigConstants {
```

Comment:
Any reason to make all these public if they're just going to be used within the Druid connector? Maybe package protected is better.